Tools/scripts/freeze_modules.py

```diff
@@ -361,22 +545,41 @@ def regen_frozen(specs, dest=MODULES_DIR):
         outfile.writelines(lines)
 
 
-def regen_makefile(frozenids, frozen):
+def regen_gitignore(modules):
+    frozenfiles = [
+        '# These are auto-generated by Tools/scripts/freeze_modules.py.',
```

Comment:
I think I said before that it feels simpler to just add a fixed pattern to .gitignore that ignores every frozen .h file.